src/ApiModules/customSuggestions.ts

```diff
@@ -1,40 +1,19 @@
 import { PointApi } from "../main";
-import { BaseMeta } from "./autocompleteSession";
+import { Snippet } from "./autocompleteSession";
 
 /** Result of a GET request to api */
 export interface GetResponse {
-  suggestions: Suggestion[];
-  hotkeys: Hotkey[];
+  snippets: Snippet[];
 }
 
 /** Result containing just a status field */
 interface StatusResponse {
   status: string;
 }
 
-/** Blacklisted suggestion object */
-export interface Blacklist {
-  id: string;
-  text: string;
-}
-
-/** Custom suggestion object. Adds custom suggestion text to the dropdown */
-export interface Suggestion {
-  id: string;
-  text: string;
-}
-
-/** Hotkey suggestion object. Autocompletes ':trigger' to 'text' */
-export interface Hotkey {
-  id: string;
-  trigger: string;
-  text: string;
-  type: "generated" | "custom"; // generated vs custom
-  labels: string[];
-}
 
 /** Class to keep track of api credentials and make requests to the custom suggestions api */
-export default class CustomSuggestionsApiModule {
+export default class SnippetApiModule {
   private readonly api: PointApi;
 
   private readonly url: string = "/extension/custom";
```

Comment:
If we could rename this endpoint to something less meaningless like /snippet (and make it more restful like) that would be fantastic although it's not really related much to snippets refactor.